.drone.yml

```diff
@@ -122,6 +122,33 @@ trigger:
   event:
   - push
 
+
+---
+kind: pipeline
+type: exec
```

Comment:
can this execute in `docker` instead of exec? could use this base image https://hub.docker.com/r/nixos/nix/ ?

Reply:
this is the same with our integration test.